other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -11,7 +11,7 @@
     "<h2 style=\"color:green\">Usage</h2>\n",
```

Comment:
<p>specify whether termination selection is done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Smth like:  Choose the termination selection mode: interactive with prompt or by index in the code  ?